index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Mobile First Project</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.3/css/all.min.css"/>
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto&display=swap" rel="stylesheet">
+</head>
+<body>
+
+    <header class="main-header">
+
+            <nav class="header">
+
+                <div class="name"><a href="#" style="color: white; text-decoration: none;">Paul Orife</a></div>
+                <div class="menu"><i class="fas fa-bars"></i></div>
```

Comment:
Kindly check your hamburger menu icon, it doesn't show up in 375px. In Figma, for mobile view, we have 375px width.
<img width="405" alt="Screenshot 2022-12-01 at 12 55 38 AM" src="https://user-images.githubusercontent.com/25887253/204917462-54baf28d-768b-40de-b382-a69136974ece.png">

But when we extend the screen you can see it is there. Please follow the Figma template for the sizes, margins and paddings.

<img width="352" alt="Screenshot 2022-12-01 at 1 10 41 AM" src="https://user-images.githubusercontent.com/25887253/204919349-4173ea3f-eea1-4fb9-be59-812c491220ad.png">